src/Serilog.Sinks.PeriodicBatching/Sinks/PeriodicBatching/PeriodicBatchingSink.cs

```diff
@@ -53,24 +58,21 @@ public abstract class PeriodicBatchingSink : ILogEventSink, IDisposable
         /// <param name="batchSizeLimit">The maximum number of events to include in a single batch.</param>
         /// <param name="period">The time to wait between checking for event batches.</param>
         protected PeriodicBatchingSink(int batchSizeLimit, TimeSpan period)
-        {
-            _batchSizeLimit = batchSizeLimit;
-            _queue = new BoundedConcurrentQueue<LogEvent>();
-            _status = new BatchedConnectionStatus(period);
-            
-            _timer = new PortableTimer(cancel => OnTick());
-        }
+            : this(batchSizeLimit, period, NoQueueLimit) { }
 
         /// <summary>
         /// Construct a sink posting to the specified database.
         /// </summary>
         /// <param name="batchSizeLimit">The maximum number of events to include in a single batch.</param>
         /// <param name="period">The time to wait between checking for event batches.</param>
-        /// <param name="queueLimit">Maximum number of events in the queue.</param>
+        /// <param name="queueLimit">Maximum number of events in the queue - use <see cref="NoQueueLimit"/> for an unbounded queue.</param>
         protected PeriodicBatchingSink(int batchSizeLimit, TimeSpan period, int queueLimit)
-            : this(batchSizeLimit, period)
         {
+            _batchSizeLimit = batchSizeLimit;
```

Comment:
Perhaps we should check this value to ensure it's greater than one, so that misconfigurations using `NoQueueLimit` here will be detected?